evaluation/feature_test.go

```diff
@@ -546,13 +546,13 @@ func TestFeatureConfig_EvaluateWithPreReqFlags(t *testing.T) {
 	onBool := Variation{
 		Name:       stringPtr("On"),
 		Value:      "true",
-		Identifier: "on",
```

Comment:
Was this change essential for the test to pass?
If so it makes me think we still have an issue somewhere where we are comparing value to identifier.

Reply:
yep, it's related to this https://harness.atlassian.net/browse/FFM-5116